features/precompile_header_with_all_headers.prf

```diff
@@ -1,17 +1,19 @@
-!macx-ios*:CONFIG += precompile_header
-PRECOMPILED_HEADER = lib$${TARGET}.h
-isEmpty(PCH_HEADERS): PCH_HEADERS = $$HEADERS
+macx {
```

Comment:
Precompiled headers should be used on all platforms. I reduces the build time significantly. I can see that this fails on Windows but does it fail on Ubuntu as well?

Use "unix" instead of "macx" to keep it for both Ubuntu, OS X and iOS.